samples/SocialWeather/PersistentConnectionLifeTimeManager.cs

```diff
@@ -23,9 +23,9 @@ public PersistentConnectionLifeTimeManager(FormatterResolver formatterResolver)
 
         public void OnConnectedAsync(ConnectionContext connection)
         {
-            connection.Features.Get<IConnectionMetadataFeature>().Metadata["groups"] = new HashSet<string>();
+            connection.Items["groups"] = new HashSet<string>();
```

Comment:
Can Items be shared with other pieces of code? Should there be a prefix on it so make it less likely to conflict with someone else who might add data with a `"groups"` key? e.g. `"socialweather-groups"`.

Also these keys should be `const` fields. It's just a sample but try to encourage good practice though leadership :)

Reply:
In general yea, but this is just a crappy sample 😄